<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloudMusic</class>
 <widget class="QMainWindow" name="CloudMusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CloudMusic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="TopWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1030</width>
      <height>75</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1030</width>
      <height>75</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1030</width>
      <height>75</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="MusicLogo">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>65</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="search">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>89</width>
         <height>69</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="setting">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>69</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="skin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>69</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="min">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>69</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="max">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_17">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>9</width>
         <height>69</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exit">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_18">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>69</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="bottomWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>635</y>
      <width>1030</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1030</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1030</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bottonWidget{
background-color:gray
};</string>
    </property>
    <widget class="QLabel" name="songe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>50</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>image</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="volume">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimumSize">
         <size>
          <width>113</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>113</width>
          <height>29</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="list">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>268</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="circulation">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="prev">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>14</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="stop">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="next">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>14</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="lyrics">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>521</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="StartTime">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>0 : 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="MediaPlayProgress">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="playerTime">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>175</width>
      <height>560</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>175</width>
      <height>560</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>175</width>
      <height>560</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>发现音乐</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="NullWidget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>845</width>
      <height>560</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>845</width>
      <height>560</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>845</width>
      <height>560</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>845</width>
       <height>560</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>845</width>
       <height>560</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>845</width>
       <height>560</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="widget">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <zorder>NullWidget</zorder>
   <zorder>TopWidget</zorder>
   <zorder>bottomWidget</zorder>
   <zorder>listWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
